types/puppetdb/ciphersuites.pp

```diff
@@ -0,0 +1,46 @@
+# Valid SSL Cipher Suites for puppetdb
+type Simp::Puppetdb::Ciphersuites = Enum[
```

Comment:
If we attempt to validate all possible values as an enumeration, it will prevent admins from configuring PuppetDB with other ciphers in the future.

Reply:
@op-ct If we don't want to do this, we need to change it in `pupmod` as well.

I'm going to push it for now and we can decide to fix both (or neither) later as a separate issue.